examples/azure/terraform-azure-actiongroup-example/variables.tf

```diff
@@ -0,0 +1,79 @@
+variable "resource_group_name" {
```

Comment:
NIT: Can you add block comments to denote which vars are required and which vars are optional? Similar to what we have in [our aws example](https://github.com/gruntwork-io/terratest/blob/master/examples/terraform-aws-example/variables.tf). It helps with readability when scanning the vars.

Reply:
Added block comments